code/obj/machinery/plantpot.dm

```diff
@@ -1264,6 +1264,29 @@ TYPEINFO(/obj/machinery/plantpot)
 					var/obj/critter/C = CROP
 					C.friends = C.friends | src.contributors
 
+				else  if (istype(CROP,/obj/item/Tumbling_Creeper))
+					// tumbling creepers behave with their DNA values are like food, but cannot be eaten.... ugh
+					// TODO: Make a proc that transfers all seed information, included hybrids
```

Comment:
todo?

Reply:
Thats mostly a generic proc that would need to be implemented in a few places, like seeds being made by eating fruits ( that part is copied from there). So i would rather make that in a separate PR than adfing here.

Reply:
I just added to proc. I won't implement it in other places in the scope of this PR (like fruits). But i think it will be usefull there as well.